cipher/Cargo.toml

```diff
@@ -14,15 +14,15 @@ categories = ["cryptography", "no-std"]
 
 [dependencies]
 crypto-common = "0.2.0-rc.1"
-inout = "0.2.0-rc.0"
```

Comment:
Actually this is a incompatible change. I just noticed that 0.2.0-rc.0 can be "cargo update" to 0.2.0-rc.3 which introduce https://github.com/RustCrypto/AEADs/pull/650

Reply:
It literally says it’s a breaking change in the commit message. These are prereleases and they frequently contain breaking changes. If you want to avoid breaking changes, use stable releases.

Reply:
Reasonable. Downgrade to aes-gcm=0.10.3 works.